source/app/pages/404.js

```diff
@@ -0,0 +1,5 @@
+import ErrorPage from '../components/ErrorPage/ErrorPage'
+
+export default function Custom404() {
+  return <ErrorPage statusCode={404} />
```

Comment:
In CloudFront , we set up a default page to go for errors. See [here](https://github.com/awslabs/document-understanding-solution/blob/master/source/lib/cdk-textract-stack.ts#L196)
Do you want to also update it here?

Reply:
I've seen the error page get used. We need the cloudfront config to use the index.html file so that the user still ends up on our app, which will recognize that it is an unrecognized route and render a 404 error.